Source/EasyNetQ/RabbitHutch.cs

```diff
@@ -133,12 +133,12 @@ public static IBus CreateBus(ConnectionConfiguration connectionConfiguration, Ac
     /// <returns>
     /// A new <see cref="RabbitBus"/> instance.
     /// </returns>
-    public static IBus CreateBus(Func<IServiceResolver, ConnectionConfiguration> connectionConfigurationFactory, Action<IServiceRegister> registerServices)
+    public static SelfHostedBus CreateBus(Func<IServiceResolver, ConnectionConfiguration> connectionConfigurationFactory, Action<IServiceRegister> registerServices)
```

Comment:
I really like that new name `SelfHostedBus`. It is much more descriptive about what's it's actually doing.
It's now clear that we are creating bus with an internal container so we have to dipose it. Much better than returning IBus with random IDisposable iherited.